src/sentry/integrations/github/client.py

```diff
@@ -71,9 +71,10 @@ def get_repo(self, repo: str) -> JSONData:
         return repository
 
     # https://docs.github.com/en/rest/git/trees#get-a-tree
-    def get_tree(self, repo_full_name: str, tree_sha: str) -> List[str]:
-        tree = []
+    def get_tree(self, repo_full_name: str, tree_sha: str) -> JSONData:
```

Comment:
Moving the logic of extracting files from a tree to `get_repo_files`, thus, return type change.